docs/new-environment.md

```diff
@@ -1,9 +1,47 @@
-# Setting up new deployment environments
+# Creating and Deploying to a new environment in Azure
 
-New deployment environments are configured by Azure specialists via the Azure YAML and JSON files.
+All the steps required to create and deploy to an Azure environment are written in the [Pipelines deploy template file](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/azure-pipelines-deploy-template.yml). This template is referenced inside the main Azure Pipelines configuration file and reused with appropriate parameters for the required environment.
 
-Once a fresh environment is deployed, it is necessary to enable feature flags
-and perhaps grant access to support and provider users.
+## Configure Variable Groups in Azure DevOps
+For deploying to a new environment, we need to configure the required variables in a variable group that will hold the values to be passed as parameters to the pipeline deployment steps.
+
+Below are the existing variables groups used in the apply pipelines.
+The groups with `APPLY - ENV` prefix hold environment specific values.
+Variable Groups |
+------------ |
+APPLY - ENV - DevOps |
+APPLY - ENV - Pentest |
+APPLY - ENV - Production |
+APPLY - ENV - QA |
+APPLY - ENV - Sandbox |
+APPLY - ENV - Staging |
+APPLY - Shared Variables |
+Docker Shared Variables |
+
+To create a new environment, clone an existing environment specific variable group and change the appropriate values.
+
+## Create a new stage in the deployment pipeline for the new environment
+
+There are two deployment stages defined in `azure-pipelines.yml`, `deploy_qa` and `deploy_devops` to deploy to QA and DevOps environment respectively. `deploy_qa` stage is run only if the build is triggered for the `master` branch and `deploy_devops` stage only if the build is triggered from a branch name defined in `devDeployBranchNameOverride` variable inside the `APPLY - ENV - DevOps` variable group.
+
+To create or deploy to a new environment, create a new stage in the pipeline by cloning an existing stage and modifying the run conditions and variables.  
+
+Make sure to change the `subscriptionPrefix` and `subscriptionName` to reflect the correct Azure Subscription to be used for the new environment.  
+
+The value used for `resourceEnvironmentName` should be used as the prefix for the host value defined inside the `authorisedHosts` variable in the variable group.  
```

Comment:
Mention the naming policy restrictions here. i.e the resource name can't only be in the format `dNN`, `tNN` or `pNN` for the development, test and production subscriptions respectively, where `NN` is a two digit number.